package.json

```diff
@@ -1,64 +1,44 @@
 {
-  "name": "richtypo",
-  "version": "0.0.0-development",
-  "description": "HTML typography enhancer for English and Russian texts",
-  "homepage": "https://github.com/sapegin/richtypo.js",
-  "author": {
-    "name": "Artem Sapegin",
-    "url": "http://sapegin.me/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/sapegin/richtypo.js.git"
-  },
-  "bugs": {
-    "url": "https://github.com/sapegin/richtypo.js/issues"
+  "private": true,
+  "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "babel-core": "^7.0.0-bridge",
+    "babel-jest": "^23.6.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-watch": "https://github.com/monathur/babel-watch.git",
+    "eslint": "^5.7.0",
+    "eslint-config-prettier": "^3.1.0",
```

Comment:
I don't think we use it now.

Reply:
Indeed, fixed in 95b57a52e6c8e2c6cad32cbc776529a690287185